jobs/github/github-check-merge.yml

```diff
@@ -122,6 +122,11 @@
           timeout: 60
           fail: true
           type: absolute
+    parameters:
+      - string:
+          default: '1.18'
```

Comment:
We should not need this here since we now extract the go version from go.mod and if we can't do that it should be an error
The `setup-go-environment` job does need GOVERSION set but we should be passing that in from go.mod